docker/dist/Dockerfile.dist.ubuntu14.04

```diff
@@ -35,7 +35,8 @@ ADD artifacts /heron
 WORKDIR /heron
 
 # run heron installer
-RUN /heron/heron-install.sh
+RUN /heron/heron-install.sh \
+    && rm -f /usr/local/heron/dist/heron-core.tar.gz
```

Comment:
should "&& rm -f /heron/heron-install.sh \" be added as well?

Reply:
It is in Line 60.